float64 x
float64 y
float64 yaw
float64 speed
float64 start_time
float64 param0
float64 param1
float64 param2
float64 rho
int32 type